daml-lf/archive/src/main/protobuf/com/daml/daml_lf_1_13/daml_lf_1.proto

```diff
@@ -912,35 +877,6 @@ message Expr {
     Expr expr = 2;
   }
 
-  // Wrap an exception value in AnyException
-  // *Available in versions >= 1.dev*
-  message ToAnyException {
-    // type of argument. Must be an exception type.
-    Type type = 1;
-    // argument
-    Expr expr = 2;
-  }
-
-  // Extract the given exception type from AnyException or return None on type-mismatch
-  // *Available in versions >= 1.dev*
-  message FromAnyException {
-    // type that should be extracted. Must be an exception type.
-    Type type = 1;
-    // Value of type AnyException
-    Expr expr = 2;
-  }
-
-  // Throw an exception.
-  // *Available in versions >= 1.dev*
-  message Throw {
-    // Overall type of the "throw" expression.
-    Type return_type = 1;
-    // Type of exception to throw. Must be an exception type.
-    Type exception_type = 2;
-    // Value of type "exception_type".
-    Expr exception_expr = 3;
-  }
-
   message Experimental {
```

Comment:
you could remove Experimental completely.

Reply:
Good point, done :)